public/diagnostic.js

```diff
@@ -23,3 +23,26 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+
+
```

Comment:
There needs to be an Order class, and inside of Order class a computed property `orderPrice`, which is price times quantity.